web/template/partial/course/manage/create-lecture-form-slides/lecture-media-slide.gohtml

```diff
@@ -2,24 +2,33 @@
     <div class="mb-4">
         <label x-show="formData.premiere || formData.vodup && !loading">
             <span class="text-sm text-5">Combined Video (mp4, if possible h264)</span>
-            <input type="file" accept="video/mp4" class="btn tl-choose-file w-full mt-2 mx-2"
-                   x-on:change="updateFiles('COMB', Object.values($event.target.files))">
+            <label class="flex">
+                <input id="fileSelection1" type="file" accept="video/mp4" class="btn tl-choose-file mt-2 mx-2 w-full"
+                       x-on:change="updateFiles('COMB', Object.values($event.target.files));">
+                <button type="button" title="Deselect File" class="fa fa-trash mt-2 mx-2" @click="updateFiles('COMB', []);document.getElementById('fileSelection1').value = '';"></button>
```

Comment:
Instead of adding an id and querying the document, can you use [x-ref](https://alpinejs.dev/directives/ref)?